quickstart-mongodbatlas/templates/mongodbatlas-cfn-quickstart.template.json

```diff
@@ -0,0 +1,355 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Deploy a MongoDB Atlas Replica Set on AWS (existing VPC).",
```

Comment:
Probably fine to just say: Deploy MongoDB Atlas  on AWS (with existing VPC)
 btw this is cool - so this is a template created for the user via the quickstart?